tox.ini

```diff
@@ -55,6 +56,33 @@ commands =
     mypy --python-version 3.5 src/build
     mypy --python-version 2.7 src/build
 
+[testenv:py27-minversion]
+description = check minimum versions required of all dependencies
+deps =
+  importlib-metadata==0.22
+  virtualenv==20.0.35
+  typing==3.5.2.2
+  pep517==0.9.1
+  packaging==19.0
+  toml==0.10.0
```

Comment:
Testing with lower version dependencies is usually done via pip constraint files. Any reason why we should not use that instead of this? I think that's more universally usable.

Reply:
Sure, can do that. As long as we don't have dependabot updating requirements.txt's or something. :)